grin/test/AbstractInterpretation/ExtendedSyntax/LiveVariableSpec.hs

```diff
@@ -747,16 +814,16 @@ spec = describe "Live Variable Analysis" $ do
             n0 <- pure (CBool z0)
             n1 <- pure (CWord z0)
             p0 <- case z0 of
-                    0 -> store n0
-                    1 -> store n1
+              0@alt0 -> store n0
```

Comment:
Do we want to have extra spaces here too?

Reply:
And at all the subsequent instances.

Reply:
Yes, I will put spaces here too.